project_task_pull_request/tests/test_project_task_pull_request.py

```diff
@@ -9,14 +9,31 @@ class TestProjectTaskPullRequest(TransactionCase):
 
     def setUp(self):
         super(TestProjectTaskPullRequest, self).setUp()
-        self.project_1 = self.env['project.project'].create({
+
+        project_1_vals = {
             'name': 'Test Project',
             'pr_required_states': [(4, 6, 0)],
-        })
-        self.project_2 = self.env['project.project'].create({
+        }
+
+        project_2_vals = {
             'name': 'Test Project',
             'pr_required_states': [(4, 6, 0), (4, 5, 0)],
-        })
+        }
+
+        module = self.env['ir.module.module'].search(
+            [('name', '=', 'project_key')])
+
+        # The 'project_key' module add a 'key' field and a
+        # '_sql_constraints' to this field.
+        # This code fix compatibility in tests when
+        # 'project_key' is installed.
+        if module.state == 'installed':
```

Comment:
Instead of this, you can try:
```python
class TestProjectTaskPullRequest(TransactionCase):
    post_install = True
    at_install = False
```
so you don't have to put conditionals for each possible module

Reply:
@pedrobaeza this approach is not working.